packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -36,13 +36,9 @@ interface BaseArguments extends CreateWorkspaceOptions {
 
 interface NoneArguments extends BaseArguments {
   stack: 'none';
-  workspaceType: 'package-based' | 'integrated';
-}
-
-interface TsArguments extends BaseArguments {
-  stack: 'ts';
-  workspaceType: 'standalone' | 'integrated';
+  workspaceType: 'package-based' | 'integrated' | 'standalone' | 'integrated';
```

Comment:
There are 2 integrated here.

What is the difference between `package-based` and `integrated`?

* `package-based` has `packages` directory while `integrated` has `libs`?
* `package-based` has `package.json` projects while `integrated has `project.json` ?
* `package-based` projects do not use executors while `integrated` do?